arch/arm/src/kinetis/Kconfig

```diff
@@ -996,6 +1003,60 @@ config KINETIS_UARTFIFOS
 	default n
 	depends on KINETIS_UART0
 
+config UART0_RXDMA
+  bool "UART0 Rx DMA"
+  default n
+  depends on KINETIS_UART0 && KINETIS_DMA
+	---help---
+		In high data rate usage, Rx DMA may eliminate Rx overrun errors
+
+config UART1_RXDMA
+  bool "UART1 Rx DMA"
+  default n
+  depends on KINETIS_UART1 && KINETIS_DMA
+	---help---
+		In high data rate usage, Rx DMA may eliminate Rx overrun errors
+
+config UART2_RXDMA
+  bool "UART2 Rx DMA"
+  default n
+  depends on KINETIS_UART2 && KINETIS_DMA
+	---help---
+		In high data rate usage, Rx DMA may eliminate Rx overrun errors
+
+config UART3_RXDMA
+  bool "UART3 Rx DMA"
+  default n
+  depends on KINETIS_UART3 && KINETIS_DMA
+	---help---
+		In high data rate usage, Rx DMA may eliminate Rx overrun errors
+
+config UART4_RXDMA
+  bool "UART4 Rx DMA"
+  default n
+  depends on KINETIS_UART4 && KINETIS_DMA
+	---help---
+		In high data rate usage, Rx DMA may eliminate Rx overrun errors
+
+config UART5_RXDMA
```

Comment:
The DMA support for LPUART has been removed since it was wrong anyways, at least in the context how RXDMA works (as it is not really using the NuttX serial DMA infrastructure).

It is annoying that there are two kinds of uart which necessitates some degree of code duplication.

I will add the correct configs back and stub it out for now.